### DISCUSSION TITLE
Using Connection Pool from Outside Resource

### DISCUSSION THREAD

**Discussion by cboecking** [PostgREST/postgrest#3728] [Q&A] [answered]
Hi Team, 

Is the connection pool available from a public url?

I am asking because I plan to create a public schema that is accessible to both external sql clients as well as postgrest. I do not want to manage two connection pools if I do not have to. Thank you for your time and efforts on this project!

Reference: https://docs.postgrest.org/en/v12/references/connection_pool.html

Chuck

**Comment by wolfgangwalther** [comment, marked as answer]
There is nothing built-in. You might want to use a connection pooler between postgresql and postgrest, so that you can re-use the same connections with other clients, too.

https://tembo.io/blog/postgres-connection-poolers

**Comment by cboecking** [comment]
Hi @wolfgangwalther - confirmed and thank you! 

For what it is worth, I am inclined to want to want to use pgcat because of its relations to pgrx and postgresml. Putting this out there in case others have opinions.
